pkg/plugins/platform/local.go

```diff
@@ -187,7 +188,8 @@ func (l *LocalApp) UnpackResources() error {
 
 	// Ensure sites/default is readable.
 	if l.GetType() == "drupal7" || l.GetType() == "drupal8" {
-		os.Chmod(path.Join(basePath, ".ddev", "files", "docroot", "sites", "default"), 0755)
+		err := os.Chmod(path.Join(basePath, ".ddev", "files", "docroot", "sites", "default"), 0755)
+		util.CheckErr(err)
```

Comment:
it feels like error should just be returned here instead of using `util.CheckErr`, since this function already has an error return value.